e2e/explorer/api/src/app/etherscan/logs.ts

```diff
@@ -0,0 +1,46 @@
+import { GetParams, Logs } from "./types";
+import { store } from '../store';
+
+export function getLogs(params: GetParams<Logs>) {
+  const { address } = params;
```

Comment:
You're using "address" only once line 30. Better remove line 5 and do that line 30: 
```typescript
if (log.address !== params.address) return false;
```